plugins/woocommerce/client/blocks/assets/js/blocks/product-specifications/block.json

```diff
@@ -17,6 +17,18 @@
 		"fontSize": {
 			"type": "string",
 			"default": "medium"
```

Comment:
As mentioned below, `fontSize` should be removed from here as it's not block's attribute, it's only injected via `useBlockProps` thanks to `supports`declaration.